alf/utils/adaptive_normalizer.py

```diff
@@ -49,19 +49,24 @@ def __init__(self,
         super(AdaptiveNormalizer, self).__init__(
             tensor_spec, norm_update_rate=self._update_ema_rate)
 
-    def normalize(self, tensors):
+    def normalize(self, tensors, clip_value=1.0):
```

Comment:
This averager is used in different places. I think default should be no clip.

Reply:
updated. But note that tf_agents does have a default clip value of 5.0 originally. Not it's set to -1.0 (ignored)